src/pages/index.js

```diff
@@ -97,3 +208,28 @@ profileEditButton.addEventListener("click", () => {
 addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
 });
+
+profileImageContainer.addEventListener("click", () => {
+  avatarPopup.open();
+  avatarFormValidator.resetValidation();
+});
+
+// Initial Data Fetch
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, initialCards]) => {
+    userInfo.setUserInfo({
+      name: userData.name,
+      job: userData.about,
+      _id: userData._id,
+    });
+
+    profileImage.src = userData.avatar;
+    cardSection.renderItems(
+      initialCards.map((card) => ({
+        ...card,
+        userId: userData._id,
+        isLiked: card.isLiked,
+      }))
+    );
+  })
+  .catch((err) => console.error(err));
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);`